h/api/db/types.py

```diff
@@ -65,6 +66,79 @@ def process_result_value(self, value, dialect):
         return _get_urlsafe_from_hex(hexstring)
 
 
+class MutableList(Mutable, list):
```

Comment:
Perhaps make a note here that we can remove this entirely when SQLAlchemy 1.1 is released in favour of `sqlalchemy.ext.mutable.MutableList`.